canvg.js

```diff
@@ -2973,6 +2973,7 @@
 				e.render(ctx);
 				if (isFirstRender) {
 					isFirstRender = false;
+					svg.stop();

```

Comment:
if animations are causing a leak, can just do `ignoreAnimation: true` and that won't start this timer